<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug Watchdog</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="debugButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="killButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Kill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QListView" name="stoppedProcessesListView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>81</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="unwatchButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Unwatch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="watchButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Watch</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="binaryToWatchText">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="watchedBinariesListView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>81</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="copyRightBanner">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>541</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Watchdog - Â© Copyright 2017 martin.uy - Licensed under GPL.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stopped Processes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Watched Binaries</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Debug Watchdog</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="onOffCheckBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="statusTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>345</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusMessage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>345</y>
      <width>491</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
